public/sass/pages/_explore.scss

```diff
@@ -55,6 +55,25 @@
     margin-top: 2 * $panel-margin;
   }
 
+  .time-series-disclaimer {
+    width: 300px;
+    margin: 10px auto;
```

Comment:
Please use variables here and for all values below as well. Check out the other classes in this file. For small values just divide a base variable.